src/main/java/cinema/model/dto/CinemaHallRequestDto.java

```diff
@@ -11,6 +11,9 @@
     @Size(min = 5, message = "Description is incomplete")
     private String description;
 
+    public CinemaHallRequestDto() {
+    }
+
     public CinemaHallRequestDto(int capacity, String description) {
```

Comment:
do you create RequestDto objects in your app?

Reply:
done